.github/workflows/publish-technical-documentation-release-mimir.yml

```diff
@@ -1,77 +1,32 @@
 jobs:
   test:
     uses: ./.github/workflows/test-docs.yml
-
   sync:
-    runs-on: "ubuntu-latest"
-    needs: "test"
+    if: github.repository == 'grafana/mimir'
+    needs: test
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: ubuntu-latest
     steps:
-
-    - name: "Checkout Mimir repo"
-      uses: "actions/checkout@v4"
-      with:
-        fetch-depth: 0
-
-    - name: "Checkout Actions library"
-      uses: "actions/checkout@v4"
-      with:
-        repository: "grafana/grafana-github-actions"
-        path: "./actions"
-
-    - name: "Install Actions from library"
-      run: "npm install --production --prefix ./actions"
-
-    - name: "Determine if there is a matching release tag"
-      id: "has-matching-release-tag"
-      uses: "./actions/has-matching-release-tag"
-      with:
-        ref_name: "${{ github.ref_name }}"
-        release_tag_regexp: "^mimir-(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)$"
-        release_branch_regexp: "^release-(0|[1-9]\\d*)\\.(0|[1-9]\\d*)$"
-
-    - name: "Determine technical documentation version"
-      if: "steps.has-matching-release-tag.outputs.bool == 'true'"
-      uses: "./actions/docs-target"
-      id: "target"
-      with:
-        ref_name: "${{ github.ref_name }}"
-
-    - name: "Clone website-sync Action"
-      if: "steps.has-matching-release-tag.outputs.bool == 'true'"
-      # WEBSITE_SYNC_TOKEN is a fine-grained GitHub Personal Access Token that expires.
-      # It must be regenerated in the grafanabot GitHub account and requires a Grafana organization
-      # GitHub administrator to update the organization secret.
-      # The IT helpdesk can update the organization secret.
-      run: "git clone --single-branch --no-tags --depth 1 -b master https://grafanabot:${{ secrets.WEBSITE_SYNC_TOKEN }}@github.com/grafana/website-sync ./.github/actions/website-sync"
-
-    - name: "Publish to website repository (release)"
-      if: "steps.has-matching-release-tag.outputs.bool == 'true'"
-      uses: "./.github/actions/website-sync"
-      id: "publish-release"
-      with:
-        repository: "grafana/website"
-        branch: "master"
-        host: "github.com"
-        # PUBLISH_TO_WEBSITE_TOKEN is a fine-grained GitHub Personal Access Token that expires.
-        # It must be regenerated in the grafanabot GitHub account and requires a Grafana organization
-        # GitHub administrator to update the organization secret.
-        # The IT helpdesk can update the organization secret.
-        github_pat: "grafanabot:${{ secrets.PUBLISH_TO_WEBSITE_TOKEN }}"
-        source_folder: "docs/sources/mimir"
-        # Append ".x" to target to produce a v<major>.<minor>.x directory.
-        target_folder: "content/docs/mimir/${{ steps.target.outputs.target }}.x"
-        # Patch releases may not have any new technical documentation.
-        allow_no_changes: true
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
```suggestion
          # Full fetch depth is required to fetch tags. The publishing workflow uses tags to prevent publishing a release branch before it has been formally released, as determined by the presence of a matching tag for the release branch.
          fetch-depth: 0
```